### PR TITLE
[IOTDB-5587] Update the cluster info and node info of metrics

### PR DESCRIPTION
**Pull request by SpriCoder**
1. Add ClusterName and NodeId into the metrics' tags, prometheus reporter and iotdb reporter
2. Remove IP and endpoint in iotdb reporter

### CODE REVIEW

File: metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfig.java [hunk 1]
```diff
@@ -57,14 +58,16 @@ public class MetricConfig {
...
-  private Integer rpcPort = 6667;
   /** The pid of iotdb instance. */
   private String pid = "";
-  /** The running system of iotdb instance */
+  /** The running system of iotdb instance. */
   private final SystemType systemType = SystemType.getSystemType();
+  /** The type of monitored node */
+  private NodeType nodeType = NodeType.CONFIGNODE;
+  /** The name of iotdb cluster. */
+  private String clusterName = "iotdb-cluster";
```
Comment (OneSizeFitsQuorum) [inline metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfig.java:68]: default?

File: metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfig.java [hunk 2]
```diff
@@ -57,14 +58,16 @@ public class MetricConfig {
...
-  /** The address of iotdb instance that is monitored. */
-  private String rpcAddress = "0.0.0.0";
-  /** The port of iotdb instance that is monitored. */
-  private Integer rpcPort = 6667;
   /** The pid of iotdb instance. */
   private String pid = "";
-  /** The running system of iotdb instance */
+  /** The running system of iotdb instance. */
   private final SystemType systemType = SystemType.getSystemType();
+  /** The type of monitored node */
```
Comment (OneSizeFitsQuorum) [inline metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfig.java:65]: .

File: docs/UserGuide/Monitor-Alert/Metric-Tool.md [hunk 3]
```diff
@@ -757,15 +757,15 @@ data source for Apache IoTDB Dashboard.
...
+| AutoGauge、Gauge | root.__system.metric.`clusterName`.`nodeId`.name.`K1=V1`...`Kn=Vn`.value                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (OneSizeFitsQuorum) [inline docs/UserGuide/Monitor-Alert/Metric-Tool.md:765]: `clusterName`.`nodeId`.`nodetype`?

Comment (SpriCoder) [reply docs/UserGuide/Monitor-Alert/Metric-Tool.md]: nodeType.nodeId?

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Pull request merged**